README.md

```diff
@@ -66,9 +69,40 @@ Base programs for basic development environment setup. Additionally, you can fin
  >``--install-extension xabikos.JavaScriptSnippets \``   
  >``--install-extension formulahendry.auto-rename-tag \``   
  >``--install-extension wayou.vscode-todo-highlight \``   
- >``--install-extension MS-vsliveshare.vsliveshare``  
+ >``--install-extension MS-vsliveshare.vsliveshare \`` \
+ >``--install-extension shd101wyy.markdown-preview-enhanced ``  
+
+ ## Optional Recommendations
+ #### Programs
+ - [DBeaver](https://dbeaver.io/) : Universal Database tool, see and edit your databases.
+ #### Terminal Enhancements
+ - [zsh](https://formulae.brew.sh/formula/zsh) : Z Shell, built on top of bash. Best when paired with a framework for configs, plugins, and themes. Config file is called ``.zshrc`` found in ``~/.zshrc``
+ - [Oh-My-Zsh](https://github.com/ohmyzsh/ohmyzsh): Community driven framework for zsh. Tons of plugins to make your terminal experience better.
+ - [jEnv](https://www.jenv.be/) : CLI to manage global and project specific JAVA versions
+ - [bat](https://github.com/sharkdp/bat) : Like the cat command but with wings - Offers syntax highlighting and other features.
+ - [fzf](https://github.com/junegunn/fzf) : interactive Unix filter for command-line that can be used with any list; files, command history, processes, hostnames, bookmarks, git commits, etc.
+ - [exa](https://the.exa.website/) : A replacement for ls command with many features
+ - [tree](https://formulae.brew.sh/formula/tree): Display directories as trees (with optional color/HTML output)
+
 
 ## Front End
 
 ## Back End
 ---
+
+## Homebrew-Cask Quick Install List
```

Comment:
Please move this section to Option Recommendations and instructions format respectively as listed below:

#### Developer Quality of Life Enhancements
- [alfred]()

##### Copy and paste in terminal to add these programs
> ``please install commands here``  
  
 ---
*additionally please remove dropbox, iterm2, visual-studio-code from the install list*